internal/webhook/external/server.go

```diff
@@ -32,8 +37,7 @@ func (s *server) Serve(ctx context.Context, l net.Listener) error {
 	logger := logging.LoggerFromContext(ctx)
 	mux := http.NewServeMux()
 
-	// TODO: Register handlers here
-	// mux.Handle("/", ...)
+	mux.Handle("POST /", http.HandlerFunc(s.refreshWarehouseHandler))
 
 	srv := &http.Server{
 		Handler:           mux,
```

Comment:
Can we cut out `mux` here and make the server use `http.HandlerFunc(s.refreshWarehouseHandler)` directly?

Reply:
+1